sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue

```diff
@@ -1,103 +1,151 @@
 <template>
-  <modal-dialog-base
-    title="Application already in progress"
-    dialogType="warning"
-    :showDialog="showDialog"
-    @dialogClosed="dialogClosed"
-  >
-    <template v-slot:content>
-      <v-container>
-        <form>
-          <p>There is already a draft of an application in progress.</p>
-          <p>Please continue your draft application or cancel it.</p>
-        </form>
-      </v-container>
-    </template>
-    <template v-slot:footer>
-      <footer-buttons
-        secondaryLabel="Close"
-        :showPrimaryButton="false"
-        @secondaryClick="dialogClosed"
-      />
-    </template>
-  </modal-dialog-base>
-  <student-page-container>
-    <v-sheet elevation="1" class="mx-auto">
-      <v-container>
-        <formio
-          formName="studentapplication"
-          :data="initialData"
-          @loaded="formLoaded"
-          @customEvent="customEventCallback"
-        ></formio>
-      </v-container>
-    </v-sheet>
-  </student-page-container>
+  <v-form ref="startApplicationForm">
+    <modal-dialog-base
+      title="Application already in progress"
+      dialogType="warning"
+      :showDialog="showDialog"
+      @dialogClosed="closeDialog"
+    >
+      <template v-slot:content>
+        <v-container>
+          <form>
+            <p>
+              There is already an application selected with this program year.
+              Please continue with that application or cancel it to start a new
+              application.
+            </p>
+          </form>
+        </v-container>
+      </template>
+      <template v-slot:footer>
+        <footer-buttons
+          primaryLabel="Close"
+          :showSecondaryButton="false"
+          @primaryClick="closeDialog"
+        />
+      </template>
+    </modal-dialog-base>
+
+    <student-page-container container-class="container-max-width ma-auto">
+      <template #header>
+        <header-navigator
+          title="Applications"
+          subTitle="Start New Application"
+          :routeLocation="{
+            name: StudentRoutesConst.STUDENT_APPLICATION_SUMMARY,
+          }"
+        >
+          <template #buttons>
+            <start-application />
+          </template>
+        </header-navigator>
+      </template>
+      <h2 class="category-header-large primary-color pb-4">
+        Apply for funding
+      </h2>
+      <content-group>
+        <div class="category-header-medium-small pa-2">
+          Financial Aid Application
+        </div>
+        <div class="pb-6 px-2">
+          Apply to see if you are applicable for provincial or federal loans and
+          grant. This is for students attending full-time or part-time
+          post-secondary studies.
+        </div>
+        <div class="category-header-medium-small pb-3 px-2">
+          Select program year
+        </div>
+        <v-select
+          v-model="programYearId"
+          :items="programYearOptions"
+          variant="solo"
```

Comment:
The other `v-select` are not using the `variant="solo"` do we need change this or do we need change the others, maybe?

Reply:
I don't know about the other ones but solo is the closest to this use case figma style:
![image](https://user-images.githubusercontent.com/78114138/199787552-5af437ca-95d4-46ff-be36-e90f4e1e4f73.png)
figma:
![image](https://user-images.githubusercontent.com/78114138/199788076-d297b986-2ab5-4a85-bfb4-243b3a9ed1fc.png)
Maybe removing the elevation will get better :) but at least the borders are closer.

Reply:
I am ok with would be close to Figma, we just need to try to make the same across the applications as much as possible.

Reply:
Actually vuetify 3 documentation doesn't even mention outlined variant: https://next.vuetifyjs.com/en/components/selects/
But I checked that outlined works and looks closer to figma anyway. Thanks!

Reply:
Yes @andrepestana-aot solo seems to be closer to figma.